tests/unittest_protocols.py

```diff
@@ -337,10 +337,10 @@ def test_assigned_stmts_match_as():
                 pass
         """
         )
-        subject: nodes.Const = assign_stmts[0].subject  # type: ignore
-        match_or: nodes.MatchOr = assign_stmts[1].pattern  # type: ignore
-        match_as_with_pattern: nodes.MatchAs = assign_stmts[2].pattern  # type: ignore
-        match_as: nodes.MatchAs = assign_stmts[3].pattern  # type: ignore
+        subject: nodes.Const = assign_stmts[0].subject  # type: ignore[index, union-attr]
+        match_or: nodes.MatchOr = assign_stmts[1].pattern  # type: ignore[index, union-attr]
+        match_as_with_pattern: nodes.MatchAs = assign_stmts[2].pattern  # type: ignore[index, union-attr]
+        match_as: nodes.MatchAs = assign_stmts[3].pattern  # type: ignore[index, union-attr]
```

Comment:
```suggestion
        subject: nodes.Const = assign_stmts[0].subject  # type: ignore[index,union-attr]
        match_or: nodes.MatchOr = assign_stmts[1].pattern  # type: ignore[index,union-attr]
        match_as_with_pattern: nodes.MatchAs = assign_stmts[2].pattern  # type: ignore[index,union-attr]
        match_as: nodes.MatchAs = assign_stmts[3].pattern  # type: ignore[index, union-attr]
```
I would recommend no spaces between error codes. Otherwise this looks like a python list.

Reply:
```suggestion
        subject: nodes.Const = assign_stmts[0].subject  # type: ignore[index,union-attr]
        match_or: nodes.MatchOr = assign_stmts[1].pattern  # type: ignore[index,union-attr]
        match_as_with_pattern: nodes.MatchAs = assign_stmts[2].pattern  # type: ignore[index,union-attr]
        match_as: nodes.MatchAs = assign_stmts[3].pattern  # type: ignore[index,union-attr]
```